app/helpers/flow_helper.rb

```diff
@@ -15,4 +15,48 @@ def flow_price(product)
     Flow.render_price_from_flow(@flow_exp, product) || '$ %' % product.price
   end
 
+  # cart total - ex: 88.99 CAD (@dux)
+  def flow_cart_total
+    return @cart_total if @cart_total
+
+    # format price
+    # go trough each element in line item and caluclutate exact price
+    # !!! add fallback code if product by sku not found
+    amount = simple_current_order.line_items.inject(0) do |total, line_item|
+      sku = line_item.variant.sku
+      flow_product = FlowCatalogCache.load_by_country_and_sku(@flow_exp, sku)
+      flow_product[:amount] * line_item.quantity + total
+    end
+
+    @cart_total = Flow.format_price(amount, @flow_exp)
+  end
+
+  # this renders link to cart with total cart price (@dux)
+  def flow_link_to_cart(text=nil)
+    text = text ? h(text) : Spree.t(:cart)
+
+    if simple_current_order.nil? || simple_current_order.item_count.zero?
+      text = '%s: (%s)' % [text, Spree.t(:empty)]
+      css_class = :empty
+    else
+      text = '%s: (%s) <span class="amount">%s</span>' % [text, simple_current_order.item_count, flow_cart_total]
+      css_class = :full
+    end
+
+    link_to text.html_safe, spree.cart_path, class: 'cart-info %s' % css_class
+  end
+
+  # get flow item from line item and shows harmonized price
```

Comment:
`harmonized` => 'localized`

Harmonization is really a separate process (https://docs.flow.io/#/module/landed cost/resource/harmonized-items) than retrieving the localized prices for items given an experience